python3/tests/test_functional.py

```diff
@@ -176,9 +178,9 @@ def test_localvariables(self):
         client = raygunprovider.RaygunSender(self.apiKey)
 
         try:
-            foo = "bar"
+            foo = "bar"  # noqa: F841
```

Comment:
F841 = unused variable

Reply:
Remove this in that case or add as comment for documentation?